addons/website_crm/views/website_templates_contactus.xml

```diff
@@ -2,13 +2,10 @@
 <odoo>
     <data>
         <template id="contactus_form" name="Contact Form (Opportunity)" inherit_id="website.contactus">
-            <xpath expr="//span[@data-for='contactus_form']" position="before">
```

Comment:
That xpath was made to mention that it was related to the `data-for` behavior, I guess we could have rename the variable `contactus_form_values` to mention `data-for`, or just kept it like that, why this change?
```xml
<t t-set="contactus_form_values" t-value="{
                'email_to': res_company.email,
                'name': request.params.get('name', ''),
                'phone': request.params.get('phone', ''),
                'email_from': request.params.get('email_from', ''),
                'company': request.params.get('company', ''),
                'subject': request.params.get('subject', ''),
            }"/>
            <span class="hidden" data-for="contactus_form" t-att-data-values="contactus_form_values"/>
```
I don't think it is really better to be put "after the declaration of the variable it is appending to" rather than "before the place where the variable is actually used" (especially given the fact we are losing the `data-for` hint, but we can live with that :grimacing:

Reply:
> I don't think it is really better to be put "after the declaration of the variable it is appending to" rather than "before the place where the variable is actually used" 

I think it really is. Same argument you used above: if the code in website changes, this will always work (or crash at installation) while before this would only crash at evaluation of the template / you might be using the variable before the data-for while it's not extended yet, etc. Then because it's really what you want to say "extend the definition of the values", at first I did not even noticed that it was an extension because of the xpath. With this change I feel you immediately understand what this is about, while before I had to go to the website code to check what's what.

But of course, it was not wrong before and it's not a **necessary** change. I just feel it is better and as it's for master, best doing it :shrug: